### DISCUSSION TITLE
Can't join discord

### DISCUSSION THREAD

**Discussion by ly-nxs** [UseInterstellar/Interstellar#446] [General]
I left the discord and now I'm trying to rejoin, but it won't let me, my username is ly.nxs if that helps

**Comment by bruhwhyamisobad** [comment]
> I left the discord and now I'm trying to rejoin, but it won't let me, my username is ly.nxs if that helps

Are you having that error that says "****Oh this is awkward.****
Discord has crashed"

**Reply by ly-nxs** [reply]
no, i fixed
